buildkite/bazelci.py

```diff
@@ -309,6 +311,16 @@
     },
 }
 
+LATEST_VERSION_PATTERN = re.compile(r"latest(-(?P<offset>\d+))?$")
+
+BUILDIFIER_RELEASE_PAGE = "https://api.github.com/repos/bazelbuild/buildtools/releases"
+
+BUILDIFIER_DEFAULT_VERSION = "latest"
+
+BUILDIFIER_DEFAULT_INPUT_FILES = ["BUILD", "*.bzl"]
+
+BUILDIFIER_DEFAULT_DOCKER_IMAGE = "gcr.io/bazel-untrusted/ubuntu1804:nojava"
```

Comment:
Can we just rename this to BUILDIFIER_DOCKER_IMAGE? It's not configurable, so it's not really a "default".